spec/operators/buffer-spec.js

```diff
@@ -3,6 +3,18 @@ var Rx = require('../../dist/cjs/Rx');
 var Observable = Rx.Observable;
 
 describe('Observable.prototype.buffer()', function () {
+  it.asDiagram('buffer')('should emit buffers that close and reopen', function () {
```

Comment:
Yes, both. Only `it.asDiagram` specs are used for `tests2png`, but both `it` and `it.asDiagram` are used for `npm test`.